.github/workflows/ci.yml

```
@@ -35,6 +35,17 @@ jobs:
     steps:
     - uses: actions/checkout@v4
 
+    - name: Clone the Shuttle main repository
+      if: ${{ github.ref == 'refs/heads/develop' }}
+      run: git clone https://github.com/shuttle-hq/shuttle.git
+
+    - name: Apply patches from Shuttle
+      if: ${{ github.ref == 'refs/heads/develop' }}
+      run: |
+        cd ./shuttle
+        scripts/apply-patches.sh
+        cd ..
```

We need to pass args to make the root path correct. We should also run it from the current directory, so that the cargo config will be picked up.
```suggestion
      run: ./shuttle/scripts/apply-patches.sh .cargo/config.toml "$(pwd)/shuttle"
```